.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["netflix-dea"]
```

Comment:
nice, a) didn't know about `.babelrc` for presets, also didn't know we had a `netflix-dea` one!